### ISSUE TITLE
Support cudnn on windows?

### ISSUE THREAD

**Issue by 7oud** [ShaoqingRen/faster_rcnn#81]
Whether the faster rcnn support cudnn on windows version? And if so which version of cudnn can be used?

**Comment by lixiang-ucas** [comment]
yes, i'm using it. The gpu compute capability needs to be higher than 3.0, the version of cudnn doesn't matter. I'm using cudnn-7.5-windows7-x64-v5.1.

**Comment by xiaoxiongli** [comment]
using cudnn will cause the LRN layer so slow, please change the engine in the prototxt

**Comment by lixiang-ucas** [comment]
how to change the engine in prototxt, i cannot find the parameters indicate whether to use cudnn.

**Comment by xiaoxiongli** [comment]
@lixiang-ucas  it is very easy to control every layer's "engine", just follow this:
  layer {
      name: "norm1"
      type: "LRN"
      bottom: "conv1"
      top: "norm1"
      lrn_param {
          local_size: 3
          alpha: 0.00005
          beta: 0.75
          norm_region: WITHIN_CHANNEL
          **engine: CAFFE**
      }
  }

now we have below 3 choice, the layer's default value is DEFAULT:

DEFAULT maybe means that if you compile caffe with CUDNN support, then the DEFAULT==CUDNN, otherwise, the DEFAULT==CAFFE.

enum Engine {
 DEFAULT = 0;
 CAFFE = 1;
 CUDNN = 2;
    }
